src/bilder/images/edxapp/templates/edxapp/xpro/common_values.yml

```diff
@@ -0,0 +1,577 @@
+MKTG_URLS: {}
+MKTG_URL_LINK_MAP: {}
+MOBILE_STORE_URLS: {}
+########################################################################################
+# Previously assumed to be unnecessary config as it duplicates code from               #
+# common.py. Unfortunately the `DOC_STORE_CONFIG` object does not propagate backwards, #
+# and so the entire structure has to be specified again                                #
+########################################################################################
+MODULESTORE:
+    default:
+        ENGINE: xmodule.modulestore.mixed.MixedModuleStore
+        OPTIONS:
+            mappings: {}
+            stores:
+            - ENGINE: xmodule.modulestore.split_mongo.split_draft.DraftVersioningModuleStore
+              NAME: split
+              DOC_STORE_CONFIG:
+                <<: *docstore_config
+              OPTIONS:
+                default_class: xmodule.hidden_module.HiddenDescriptor
+                fs_root: /edx/var/edxapp/data
+                render_template: common.djangoapps.edxmako.shortcuts.render_to_string
+            - ENGINE: xmodule.modulestore.mongo.DraftMongoModuleStore
+              NAME: draft
+              DOC_STORE_CONFIG:
+                <<: *docstore_config
+              OPTIONS:
+                default_class: xmodule.hidden_module.HiddenDescriptor
+                fs_root: /edx/var/edxapp/data
+                render_template: common.djangoapps.edxmako.shortcuts.render_to_string
+ORA2_FILE_PREFIX: xpro/ora2  # MODIFIED
+PARTNER_SUPPORT_EMAIL: ''
+PASSWORD_POLICY_COMPLIANCE_ROLLOUT_CONFIG:
+    ENFORCE_COMPLIANCE_ON_LOGIN: false
+PASSWORD_RESET_SUPPORT_LINK: ''
+PAYMENT_SUPPORT_EMAIL: {{ key "edxapp/sender-email-address" }}  # MODIFIED
+PLATFORM_DESCRIPTION: MIT xPRO # MODIFIED
+PLATFORM_FACEBOOK_ACCOUNT: http://www.facebook.com/YourPlatformFacebookAccount
+PLATFORM_NAME: MIT xPRO  # MODIFIED
+PLATFORM_TWITTER_ACCOUNT: '@YourPlatformTwitterAccount'
+POLICY_CHANGE_GRADES_ROUTING_KEY: edx.lms.core.default
+PRESS_EMAIL: support@xpro.mit.edu # MODIFIED
+PROCTORING_BACKENDS:
+    DEFAULT: 'proctortrack'
+    'proctortrack':
+      {{ with secret "secret-xpro/edxapp" }}
+      'client_id': '{{ .Data.proctortrack_client_id }}'
+      'client_secret': '{{ .Data.proctortrack_client_secret }}'
+      {{ end }}
+      'base_url': '{{ key "edxapp/proctortrack-base-url" }}'
+    'null': {}
+
+PROCTORING_SETTINGS: {}
+REGISTRATION_EXTRA_FIELDS:
+    city: hidden
+    confirm_email: hidden
+    country: hidden
+    gender: optional
+    goals: optional
+    honor_code: hidden
+    level_of_education: optional
+    mailing_address: hidden
+    terms_of_service: hidden
+    year_of_birth: optional
+RETIRED_EMAIL_DOMAIN: retired.invalid
+RETIRED_EMAIL_PREFIX: retired__user_
+RETIRED_USERNAME_PREFIX: retired__user_
+RETIRED_USER_SALTS:
+- OVERRIDE ME WITH A RANDOM VALUE
+- ROTATE SALTS BY APPENDING NEW VALUES
```

Comment:
should we set this?

Reply:
I'm not sure. I think this is for use by the GDPR user deletion feature. That will require some investigation.